docs/source/notebooks/mmm/mmm_lift_test.ipynb

```diff
@@ -0,0 +1,1716 @@
+{
```

Comment:
<p>Can you put the legend outside the plot (<a href="https://stackoverflow.com/questions/4700614/how-to-put-the-legend-outside-the-plot" target="_blank">https://stackoverflow.com/questions/4700614/how-to-put-the-legend-outside-the-plot</a>) and add a title?</p><p> </p><p>Also what does "column value" mean (maybe use the same terminology as the mmm example notebook)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/590/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Removed xlabel and added a descriptive title</p>